samples/Net9FunctionApp/HelloHttp.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Net9FunctionApp
+{
+    public class HelloHttp(ILogger<HelloHttp> logger)
+    {
+        [Function("HelloHttp")]
+        public IActionResult Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequest req)
```

Comment:
I know this is everywhere in the samples but as a best practice we should keep `AuthorizationLevel` as `Function` here to set the correct example for users. I remember mathewc told us for [sdk type bindings](https://github.com/Azure/azure-functions-dotnet-worker/blob/21ce5a7b653a26fe55c1e9d0bfa4a614426e1ac9/samples/Extensions/Blob/BlobInputBindingSamples.cs#L35) samples.

Reply:
Do you know why it is a best practice?